actions/build_docs/action.yml

```diff
@@ -1,14 +1,20 @@
 name: Test Documentation
 inputs:
   target:
-    required: true
     type: string
 runs:
   using: "composite"
   steps:
-    - name: Give Permission
+    - name: Give Permission to build_docs.sh
       run: chmod +x ${{github.action_path}}/build_docs.sh 
       shell: bash 
     - name: Build Documentation
+      if: ${{ github.event.repository.name != '.github'}}
```

Comment:
As we grow (particularly in the Python direction) we may run into other repos that don't need to run Doxygen in order to generate documentation. My suggestion would be to add an input option `skip_doxygen` which defaults to false instead of hard-coding repo names.